Changelog.md

```diff
@@ -1,5 +1,58 @@
 # Changelog
 
+### 1.3.0
+
+Image: 
+- registry.k8s.io/ingress-nginx/controller:v1.3.0@sha256:d1707ca76d3b044ab8a28277a2466a02100ee9f58a86af1535a3edf9323ea1b5
+- registry.k8s.io/ingress-nginx/controller-chroot:v1.3.0@sha256:0fcb91216a22aae43b374fc2e6a03b8afe9e8c78cbf07a09d75636dc4ea3c191
+
+_IMPORTANT CHANGES:_ 
+* This release removes support for Kubernetes v1.19.0
```

Comment:
Should we add the new privileges that are needed now? for 1.24 update?

Reply:
I think it is necessary to remind users

Reply:
what new privileges? for leader election?

Reply:
yes